.rubocop.yml

```diff
@@ -88,3 +88,6 @@ Metrics/PerceivedComplexity:
 
 Style/CaseEquality:
   Enabled: false
+
+Style/GueardClause:
```

Comment:
`GueardClause`?
```suggestion
Style/GuardClause:
```

Reply:
Oh Yea!
This guard is killing me. If i put all on one line it is too long.
If i break it up it wants a proper if statement.
If i give it a proper i statement, this one goes crazy.